### DISCUSSION TITLE
🙋How to match the database schema version with the app version? [Hoe bepaal ik welke versie van het database schema bij de app versie hoort]?

### DISCUSSION THREAD

**Discussion by TweakM** [dsmrreader/dsmr-reader#1741] [Questions & answers / How-to's] [answered]
### Description

N.B. Dutch or English are both fine, I'm including my question in both languages for the community. 

My Raspberry Pi SD card crashed, so I lost my installation of dsmr-reader. Fortunately, the database is on a separate machine and it remains running.
I don't know what version of the app was running when the Pi crashed. My questions are:
1. Does the app autodetect the version of the database schema and update accordingly ("migrations")?
2. If not, what is the easiest way to determine the version of the database schema and the version of the app that should go with it?
My concern is that if I don't match the app version with the correct database schema the data will become corrupt or the app will have issues later on.
Thanks for any insights!
------
Mijn Raspberry Pi SD kaart is gecrasht, daardoor ben ik mijn installatie van dsmr-reader kwijtgeraakt. Gelukkig draait de database op een aparte machine en loopt die nog gewoon.
Ik weet niet welke versie van de app draaide toen de Pi crashte. Mijn vragen zijn:
1. Detecteert de app automatisch de versie van het database schema en update die vanzelf ("migraties")?
2. Zo niet, wat is de eenvoudigste manier om de versie van het database schema en de versie van de app die daarbij hoort te bepalen?
Mijn zorg is dat als de app versie en het database schema qua versie niet bijelkaar horen dat de data corrupt raakt of dat de app later problemen krijgt.
Bedankt voor alle hulp!

### DSMR-reader version

Unknown / Onbekend

### DSMR-reader platform

Native (e.g. manual installation)

**Comment by dennissiemensma** [comment, marked as answer]
Zolang je dezelfde of een nieuwere versie installeert wordt de database automatisch bijgewerkt. De laatste versie zou dus altijd moeten werken.

**Reply by TweakM** [reply]
Top, thanks!
